diff --git a/node_modules/@whiskeysockets/baileys/lib/Utils/chat-utils.js b/node_modules/@whiskeysockets/baileys/lib/Utils/chat-utils.js
index 2eebb4e..bdcb9ef 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Utils/chat-utils.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Utils/chat-utils.js
@@ -454,7 +454,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 privacySettingDisableLinkPreviewsAction: mod.disableLinkPreviews || {}
             },
             index: ['setting_disableLinkPreviews'],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 8,
             operation: OP.SET
         };
@@ -511,7 +511,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 }
             },
             index: ['quick_reply', mod.quickReply.timestamp || String(Math.floor(Date.now() / 1000))],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 2,
             operation: OP.SET
         };
@@ -527,7 +527,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 }
             },
             index: ['label_edit', mod.addLabel.id],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 3,
             operation: OP.SET
         };
@@ -540,7 +540,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 }
             },
             index: [LabelAssociationType.Chat, mod.addChatLabel.labelId, jid],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 3,
             operation: OP.SET
         };
@@ -553,7 +553,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 }
             },
             index: [LabelAssociationType.Chat, mod.removeChatLabel.labelId, jid],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 3,
             operation: OP.SET
         };
@@ -566,7 +566,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 }
             },
             index: [LabelAssociationType.Message, mod.addMessageLabel.labelId, jid, mod.addMessageLabel.messageId, '0', '0'],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 3,
             operation: OP.SET
         };
@@ -586,7 +586,7 @@ export const chatModificationToAppPatch = (mod, jid) => {
                 '0',
                 '0'
             ],
-            type: 'regular',
+            type: 'regular_low',
             apiVersion: 3,
             operation: OP.SET
         };
